sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -2591,3 +2591,16 @@ UPDATE categories SET codes='LOINC:42348-3' WHERE name='Advance Directive' AND c
 #IfNotRow2D preference_value_sets loinc_code 95541-9 answer_code 314433002
 INSERT INTO preference_value_sets(loinc_code,answer_code,answer_system,answer_display,sort_order,active) VALUES('95541-9', 314433002, 'http://snomed.info/sct', 'Preference for health professional (finding)', 1, 1);
 #EndIf
+
+-- --------------------------------------------------------- sjp 11/20/2025 ----------------------------------------------------------------------------------------------------
+#IfColumn form_history_sdoh pregnancy_gravida
+ALTER TABLE form_history_sdoh DROP COLUMN `pregnancy_gravida`, DROP COLUMN `pregnancy_para`;
+#EndIf
+
+#IfCareTeamsV1MigrationNeeded
+-- we don't want to destroy any patient data entered in these deprecated fields but we do want to stop them from appearing on the DEM form
+UPDATE layout_options SET uor=0 WHERE form_id='DEM' AND field_id IN ('care_team_facility', 'care_team_provider', 'care_team_status') AND uor=1;
```

Comment:
conflict was down here and decided to look ahead and tuck the sql inside the ifcareteamsv1 block like you do later

Reply:
okay sounds good as long as next and last concerning sql upgrade I believe, doesn't conflict.